app/models/user.rb

```diff
@@ -145,9 +145,8 @@ def self.ownership_request_notifiable_owners
   end
 
   def self.normalize_email(email)
-    email.to_s.gsub(/\s+/, "")
-  rescue ArgumentError => e
-    Rails.error.report(e, handled: true)
+    email.to_s.encode(Encoding::ASCII_8BIT).gsub(/\s+/, "")
+  rescue Encoding::UndefinedConversionError, Encoding::InvalidByteSequenceError
```

Comment:
do we need to add a validation error here so users know what happened? if I'm reading this right, signing up with a unicode email will get a "can't be blank" validation error, which seems confusing.

Reply:
Maybe this is an argument for using default replace behavior and then the � will trigger the format error.

Reply:
Let's follow up with this later. I'm just aiming to remove the reporting and increase test coverage (so no one mistakenly changes the behavior in the future if they attempt this fix)

Reply:
Sounds good.